articles/active-directory/saas-apps/hub-planner-tutorial.md

```diff
@@ -133,9 +133,27 @@ In this section, you'll enable B.Simon to use Azure single sign-on by granting a
 
 To configure single sign-on on **Hub Planner** side, you need to send the downloaded **Certificate (Base64)** and appropriate copied URLs from Azure portal to [Hub Planner support team](mailto:hello@hubplanner.com). They set this setting to have the SAML SSO connection set properly on both sides.
 
+### Install Extension on Hub Planner.
+To enable the SSO functionality you need to first enable the extension. As the account owner or someone with rights follow these steps.
+
+1. Navigate to Settings
+1. Choose Manage Extensions - > Add/Remove Extensions from the sidebar.
+1. Find the extension for Single Sign On and Add or Try Free.
+1. Agree to T&C in the pop up that appears and click Add Now.
+
+### Enable SSO.
+Once the extension is enabled you now need to enable SSO for your account. 
+
+1. Navigate to Settings
+1. Choose "Authentication" from the sidebar.
+1. Enable SSO
+1. Add the 3 required fields and click save.
+
+![SSO Settings Link](https://hubplanner-static.s3-eu-west-1.amazonaws.com/img/extensions/sso/hub_planner_sso_config.png)
```

Comment:
Sure, but this is on your side right you need to add the image?

Reply:
Yes, we need the screenshot in the form of an image file that is included in the pull request, and then referenced from within the article. However, the content owners @v-nagta @jeevansd need to create a /media/ folder for the article, or indicate what media folder an image could be added to.